data/json/effects.json

```diff
@@ -823,15 +823,8 @@
             "speed_mod": [-2]
         },
         "scaling_mods": {
-            "speed_mod": [-7],
-            "int_mod": [-0.5],
```

Comment:
the cold was dangerous before because it sliwed you down, took away your ability to function.  these supporting penalties added up per body part.